### PR TITLE
Filters for Calculate Billing Month

### PR DESCRIPTION
**Pull request by aaronk**
Add search and an error filter for Calculate Billing Month page.
Also, ensure correct display of processing error messages

### CODE REVIEW

File: src/components/billingRecord/IFXCalculateBillingMonth.vue [hunk 1]
```diff
@@ -139,25 +200,20 @@ export default {
...
-      }
-    }
-  },
   mounted() {
     if (this.month && this.year) {
       this.selectedDate = `${this.year}-${this.month}`
+    } else {
+      this.selectedDate = this.$api.storage.getItem(this.selectedDateKey, 'session') || null
     }
+    this.search = this.$api.storage.getItem(this.searchStorageKey, 'session') || ''
```
Comment (luthian) [inline src/components/billingRecord/IFXCalculateBillingMonth.vue:209]: One thing to check; with `NICE`, if I deleted the search string, it wrote `"null"` into the session storage. When the page reloaded, it was searching for the literal string "null" which usually returned nothing.

File: src/components/billingRecord/IFXCalculateBillingMonth.vue [hunk 2]
```diff
@@ -139,25 +200,20 @@ export default {
...
+    this.onlyErrors = this.$api.storage.getItem(this.searchStorageKey, 'session') === 'true'
     this.localOrganization = this.organization
     this.setFacility()
       .then(() => {
         this.isLoading = false
         if (this.selectedDate) {
           this.getUsages()
-            .then(() => this.calculateBillingMonth())
```
Comment (luthian) [inline src/components/billingRecord/IFXCalculateBillingMonth.vue:159]: I assume we don't need to calculate the billing month here.

Comment (aaronk) [reply src/components/billingRecord/IFXCalculateBillingMonth.vue:159]: Right.  I changed the behavior a bit.  Once the year-month could come in from the session storage, it didn't make sense to trigger the calculation this way.  Can be handled with the calculator button.

File: src/components/billingRecord/IFXCalculateBillingMonth.vue (hunk 1 above)
Comment (aaronk) [reply src/components/billingRecord/IFXCalculateBillingMonth.vue:209]: IFXAPI storage takes care of this.

**Pull request merged**